backend/src/main/java/com/capstone/moneytree/service/impl/DefaultStockService.java

```diff
@@ -20,6 +21,8 @@
 @Transactional
 public class DefaultStockService implements StockService {
 
+    private static final String STOCK_NOT_FOUND = "The requested stock was not found";
```

Comment:
I think this line does not belong here. Add this into the [ExceptionMessage](https://github.com/DPigeon/Money-Tree/blob/dev/backend/src/main/java/com/capstone/moneytree/handler/ExceptionMessage.java#L4) class.